playbooks/operationalizing/how-full-is-my-cluster.adoc

```diff
@@ -0,0 +1,150 @@
+---
+---
+= How Full is my OpenShift Cluster?
+Raffaele Spazzoli <rspazzol@redhat.com>
+v1.0, 2017-08-28
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+
+= Building a Capacity Management Strategy for OpenShift?
```

Comment:
Move this line to replace line 3. We don't want 2 headers. Also, remove '?'

Reply:
done